helpers/deploymentConfig.ts

```diff
@@ -5322,6 +5332,160 @@ export const globalConfig: NetworkConfig = {
     ],
     preconfiguredAddresses: preconfiguredAddresses.opsepolia,
   },
+  opmainnet: {
+    tokensConfig: [
+      {
+        isMock: false,
+        name: "USD Coin",
+        symbol: "USDC",
+        decimals: 6,
+        tokenAddress: "0x0b2c639c533813f4aa9d7837caf62653d097ff85",
```

Comment:
Why we have this address `0x7F5c764cBc14f9669B88837ca1490cCa17c31607` of USDC in doc?